src/index.js

```diff
@@ -25,6 +26,11 @@ const schema = {
       description:
         "Possible options: none and prettier (requires prettier package installed). Defaults to prettier if `prettier` module can be resolved",
       enum: ["prettier", "none"]
+    },
+    check: {
+      description:
+        "Set to `true` to validate generated d.ts files and fail if an update is needed. Defaults to `false`. ",
```

Comment:
Minor nit: As the setting has a `boolean` type, the "Set to `true`" sentence is redundant.
```suggestion
        "Validate generated d.ts files and fail if an update is needed. Defaults to `false`. ",
```

Reply:
Thanks, fixed. I am still waiting feedbacks from maintainers to add tests & finish the PR.